src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -122,15 +126,28 @@ public function overviewAction()
         }
 
         $serviceObjects = [];
+        $productionEntityEnabled = [];
+        $privacyOK = [];
         foreach ($services as $service) {
+            $productionEntityEnabled[] = $service->isProductionEntitiesEnabled();
             $entityList = $this->entityService->getEntityListForService($service);
             $serviceObjects[] = Service::fromService($service, $entityList, $this->router);
+            $privacyOK[] = $this->serviceStatusService->hasPrivacyQuestions($service);
         }
         $serviceList = new ServiceList($serviceObjects);
 
+        // Try to get a published entity from the session, if there is one, we just published an entity and might need
+        // to display the oidc confirmation popup.
+        /** @var Entity $publishedEntity */
+        $publishedEntity = $this->get('session')->get('published.entity.clone');
+
         return $this->render('DashboardBundle:Service:overview.html.twig', [
             'services' => $serviceList,
-            'isAdmin' => false
+            'isAdmin' => false,
+            'publishedEntity' => $publishedEntity,
+            'showOidcPopup' => $this->showOidcPopup($publishedEntity),
+            'productionEntitiesEnabled' => $productionEntityEnabled,
+            'privacyStatusEntities' => $privacyOK,
```

Comment:
Not wrapped in array here, but on the next actionMethod, the PrivacyOK var is wrapped in an array. Why the difference between the two?

Reply:
nothing, this is already an array, initialized on line 130.  I changed it so the admin action also has the initialization higher up, making the two similar.